src/components/markdown.tsx

```diff
@@ -18,10 +20,21 @@ const Code = ({
     highlightElement(ref.current);
   }, [children, className]);
 
+  const handleCopy = () => {
+    if (typeof children === 'string') {
+      navigator.clipboard.writeText(children);
+    }
+  };
+
   return (
-    <code ref={ref} className={className}>
-      {children}
-    </code>
+    <div className="code-container">
```

Comment:
You don't need to use custom defined classes. LEave a div and add Tailwind already added utility classes.